serialbox/install.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -x
+set -e
+
+repo="https://github.com/ai2cm/serialbox"
```

Comment:
I think we can use upstream. Not sure if there are any local modfis anymore tbh. Maybe something to discuss at team meeting?

Reply:
I just think in our project we should have our fork (that at least does not jump unexpected), If there is no need for our fork anymore, we can just tag a version on gridtools and close ours. But as long as we have it it seems like the thing to install?

Reply:
Fine.